changelog.md

```diff
@@ -1,7 +1,7 @@
 2.8.0
 -----
 
-* FEATURE: Added `config.before_send`. Provide a lambda or proc to this config setting, which will be `call`ed when before sending an event to Sentry. Receives `event` and `hint` as parameter. `hint` is a has {:exception => ex | nil, :message => message | nil}. [@hazat, #847]
+* FEATURE: Added `config.before_send`. Provide a lambda or proc to this config setting, which will be `call`ed when before sending an event to Sentry. Receives `event` and `hint` as parameter. `hint` is a has {:exception => ex | nil, :message => message | nil}. [@hazat, #882]
```

Comment:
While we're here, can we fix some other things?

1. `when before` ➡️ `before`
1. `is a has` ➡️ `is a hash`

Reply:
@rspeicher I fixed at dd7937f